domains/200anxy.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "200anxy",
+        "email": "aadarshnair.p@gmail.com"
+    },
+    "record": {
+        "TXT": "hi"
```

Comment:
TXT record alone doesn't do anything.

Reply:
Hi I didn't really understand about what this part was, it said to choose this thing, What are we actually supposed to do?